robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenuItem.java

```diff
@@ -30,8 +31,14 @@
   private View actionView;
   private OnActionExpandListener actionExpandListener;
   private int order;
+  private Context context;
 
   public RoboMenuItem() {
+    this(RuntimeEnvironment.application);
```

Comment:
There are a lot of breaking changes between 3.0-3.1, so I don't think it matters. We're planning a regular release cadence and deprecation policy soon though.